### PR TITLE
Change delete function in backend

### PR DESCRIPTION
**Pull request by roabram**
Add delete function, CSS Animation and some basic stylings

### CODE REVIEW

**Review by Niklas-Schaefers — APPROVED** [review]
🚀 lgtm

File: README.md [hunk 1]
```diff
@@ -1,99 +1,50 @@
...
-
-We can maybe use [SemVer](http://semver.org/) for versioning. For the versions available, see the [link to tags on this repository](/tags).
-
-## Configuration
+```
+npm start
+```
 
-Here you should write what are all of the configurations a user can enter when using the project.
+Runs the server in production mode and serve production bundle from npm run build
```
Comment (SGierszewski) [inline README.md:21]:
```suggestion
Runs the server in production mode and serves production bundle from npm run build
```

File: src/app/components/Icons/DeleteIcon/DeleteIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styles from './DeleteIcon.module.css';
+
+function DeleteIcon(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+  return (
+    <svg
+      className={styles.icon}
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (SGierszewski) [inline src/app/components/Icons/DeleteIcon/DeleteIcon.tsx:8]: this is probably not needed...

File: src/app/components/Icons/TradeInIcon/TradeInIcon.tsx [hunk 3]
```diff
@@ -46,14 +46,14 @@ function TradeInIcon(props: React.SVGProps<SVGSVGElement>): JSX.Element {
           </g>
         </g>
         <text
-          id="tradeIn"
+          id="trade In"
```
Comment (SGierszewski) [inline src/app/components/Icons/TradeInIcon/TradeInIcon.tsx:49]: probably not needed as well

**Review by SGierszewski — COMMENTED** [review]
LGTM 🚀

**Pull request merged**